cmd/dynamic-rp/dynamicrp-dev.yaml

```diff
@@ -0,0 +1,43 @@
+# This is an example of configuration file.
+environment:
+  name: Dev
+  roleLocation: "global"
+storageProvider:
+  provider: "apiserver"
+  apiserver:
+    context: ''
+    namespace: 'radius-testing'
+queueProvider:
+  provider: "apiserver"
+  name: dynamic-rp
+  apiserver:
+    context: ''
+    namespace: 'radius-testing'
+secretProvider:
+  provider: "kubernetes"
+profilerProvider:
+  enabled: false
+  port: 6060
+metricsProvider:
+  prometheus:
+    enabled: false
+    path: "/metrics"
+    port: 9090
+server:
+  host: "0.0.0.0"
+  port: 5443
```

Comment:
Isn't this the app-rp's port also?

Reply:
Going to go with 8082 to follow the convention other components use.